proto/anki/stats.proto

```diff
@@ -65,6 +65,7 @@ message CardStatsResponse {
   string preset = 21;
   optional string original_deck = 22;
   optional float desired_retention = 23;
+  repeated float fsrs_params = 24;
```

Comment:
What's the purpose of this change?

Reply:
The forgetting curve displayed in the Card Info needs to read the decay (w[20]) from the parameters.